client/js/views/Header.js

```diff
@@ -11,12 +11,13 @@ module.exports = Object.create( Object.assign( {}, require('./__proto__'), {
     model: require('../models/Header'),
 
     name: 'Header',
-
-    onNavListClick(e) {
-        const target = e.target
-        if( target.tagName !== 'SPAN' ) return
-            
-        this.emit( 'navigate', `/${target.textContent.toLowerCase()}` )
+    onNavListClick(event) {
+        this.emit('navigate', `/${ event.target.textContent.split(' ').join('') }`)
```

Comment:
Not sure what
```
event.target.textContent.split(' ').join('')
```
is doing.

Reply:
This should return the name of the file to navigate to.  For example, if the user clicks the "About Us" link, `event.target.textContext.split(' ').join('')` should return "AboutUs", so the About Us page would be navigated to.  Is there a better way of doing this?

Reply:
Ahh, right, sorry I misread the code.

While I like camel casing, let's move to a more common Url format.

`event.target.textContent.replace(/s+/,'').toLowerCase()`